paypal.js

```diff
@@ -92,6 +93,11 @@ var detail_q = async.queue(function(task, next) {
     }
 
     task.COUNTRYCODE = transaction.COUNTRYCODE;
+    task.settle_amount = transaction.AMT;
+    if (task.CURRENCYCODE && task.CURRENCYCODE !== 'USD' && transaction.EXCHANGERATE) {
+      task.settle_amount = (transaction.AMT * transaction.EXCHANGERATE * 100 | 0 ) / 100;
```

Comment:
Stripping the decimal point, so that it only has two decimal points when the calculation is done